### PR TITLE
Pods 5670 - Removing twirl

### PR DESCRIPTION
**Pull request by madhuranakate**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -44,6 +44,7 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
   lazy val lockOnSchemeUrl: String = s"$migrationUrl${config.get[String](path = "urls.lockOnScheme")}"
   lazy val dataCacheUrl: String = s"$migrationUrl${config.get[String](path = "urls.dataCache")}"
   lazy val addressLookUp = s"${servicesConfig.baseUrl("address-lookup")}"
+  lazy val yourPensionSchemesUrl: String = loadConfig("urls.yourPensionSchemes")
```
Comment (grantrobson) [inline app/config/AppConfig.scala:47]: You need to update app config base for this

File: test/base/SpecBase.scala [hunk 2]
```diff
@@ -17,58 +17,68 @@
...
 
 trait SpecBase
   extends PlaySpec
     with GuiceOneAppPerSuite {
 
-  def modules(mutableFakeDataRetrievalAction: MutableFakeDataRetrievalAction): Seq[GuiceableModule] =
-    Seq(
-      bind[AuthAction].toInstance(FakeAuthAction),
-      bind[DataRetrievalAction].toInstance(mutableFakeDataRetrievalAction)
+//  def modules(mutableFakeDataRetrievalAction: MutableFakeDataRetrievalAction): Seq[GuiceableModule] =
```
Comment (grantrobson) [inline test/base/SpecBase.scala:43]: Remove commented code

**Review by grantrobson — APPROVED** [review]

File: test/base/SpecBase.scala (hunk 2 above)
Comment (madhuranakate) [reply test/base/SpecBase.scala]: done

File: app/config/AppConfig.scala (hunk 1 above)
Comment (madhuranakate) [reply app/config/AppConfig.scala:47]: on it

**Pull request merged**